src/pages/join/volunteering.js

```diff
@@ -20,6 +21,43 @@ const TitleHeading = styled.h1`
   `}
 `;
 
+const MyBtn = styled.button`
+    background-color: rgba(34,34,34,1);
+    border-color: rgba(34,34,34,1);
+    color: rgba(238,238,238,1);
+    width: 30%;
+    line-height: 1.5;
+    font-size: 1.2vw;
```

Comment:
the text is really small on mobile or smaller screens. maybe we can add a media query to change this. if you find a good solution, can you add it to the job board button as well?